packages/code-block/partials/hidden-copy-content/index.js

```diff
@@ -8,6 +8,19 @@ import { forwardRef } from 'react'
  * line numbers and so on in other parts of code-block.
  */
 function HiddenCopyContent({ code }, copyRef) {
+  const [isClient, setIsClient] = useState(false)
+
+  useEffect(() => {
+    setIsClient(true)
+  }, [])
```

Comment:
General Q: would this accomplish the same end result?

```suggestion
  if (typeof window === "undefined") return null
```

Reply:
Nope, because the initial client render would then be different from the server render and we could still encounter hydration issues.

Reply:
https://beta.reactjs.org/apis/react-dom/hydrate#handling-different-client-and-server-content